devops/ansible-role-girder/tasks/main.yml

```diff
@@ -23,6 +23,14 @@
 - import_tasks: nodejs.yml
   when: girder_web|bool
 
+- name: Upgrade pip in the virtualenv
+  pip:
+    name: pip
+    extra_args: "--update"
```

Comment:
This should use `state:`.

Reply:
Is there a particular minimum version of pip that's necessary? I'd rather ensure we have `>=` that version, rather than the latest, as that impacts idempotency.

Reply:
`cryptography` appears to need pip 21.0.1 because it needs rust.  However, it is probably dependent on the version of Python, as if `cryptography` publishes a wheel for that version, then pip 9 might be sufficient.